catalog.html

```diff
@@ -161,56 +155,56 @@ <h2 class="catalog-sorting visually-hidden">Сортирование</h2>
       </a>
     </section>
     <section class="catalog-list-section">
-      <h2 class="catalog-list visually-hidden">Список гостиниц</h2>
+      <h2 class="visually-hidden">Список гостиниц</h2>
       <div class="hotel">
         <a class="hotel-link" href="#">
           <img class="hotel-image" src="" alt="Amara Resort & Spa">
           <h3 class="hotel-title">Amara Resort & Spa</h3>
-          <img class="hotel-rating" src="" alt="Четыре звезды">
+          <span class="visually-hidden">Четыре звезды</span>
```

Comment:
Хорошо, но как-то же надо будет стилизовать звезды?

```suggestion
          <p class="stars stars-4">
            <span class="visually-hidden">Четыре звезды</span>
          </p>
```